Documentation/gettingstarted/k8s-install-default.rst

```diff
@@ -228,31 +228,28 @@ You can install Cilium on any Kubernetes cluster. Pick one of the options below:
            cilium install |CHART_VERSION|
 
     .. group-tab:: AKS
+       
+       .. note::
+
+          On AKS, Cilium can be installed either manually by administrators (Bring your own CNI) or 
+          automatically by AKS (Azure CNI Powered by Cilium). Bring your own CNI offers more flexibility 
+          and customization as administrators have full control over the installation, but does not 
+          integrate natively with the Azure network stack and administrators need to handle Cilium upgrades. 
+          Azure CNI Powered by Cilium integrates natively with the Azure network stack and upgrades are 
+          handled by AKS, but does not offer as much flexibility and customization as it is controlled by AKS. 
+          The following instructions assume Bring your own CNI. For Azure CNI Powered by 
+          Cilium, see the external installer guide :ref:`aks_install` for dedicated instructions.
 
+       .. include:: ../installation/requirements-aks.rst
```

Comment:
Why move the include statement here? I think it makes sense to keep it under the `Install Cilium` header.

Reply:
i removed legacy ipam and made instructions only for byocni remain as per this discussion https://github.com/cilium/cilium/pull/27199#pullrequestreview-1561391819 .. so no need for separate file and everything can be moved to requirements-aks.rst as like eks/gke

Reply:
Ah ok that makes sense, thanks!